### PR TITLE
AppNexus connector with Prebid

### PR DESCRIPTION
**Pull request by pablovegau**
# Background

As current AppNexus connector does not support PREBID, a new connector is needed.

# Goal

Based on the current OpenAds AppNexus connector, provide a new connector supporting PREBID.

# Implementation

- A new prebid client has been added obtaining pbjs from window
- AstClient has been refactored to obtain apntag from window
- Debounce is no longer done inside astclient but in the connector

# Further considerations

- apntag and pbjs are not included as a dependency, so they must be provided as a tag script in the page.

# Checklist

- [x] The PR relates to *only* one subject with a clear title.
- [x] I have performed a self-review of my own code
- [x] Wrote [good commit messages](http://chris.beams.io/posts/git-commit/)
- [x] My code is readable by someone else, and I commented the hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works

# Visual description

### CODE REVIEW

File: src/openads-appnexus/AppNexusConnector.js [hunk 1]
```diff
@@ -6,103 +6,208 @@ import {
...
           this._astClient.modifyTag({
-            targetId: domElementId,
-            data: updateData
+            targetId: tag.id,
+            data: tag.data
+          })
+        )
+        if (normalizedInputs.adUnits.length > 0) {
+          this._prebidClient.requestBids({
+            requestObj: {
```
Comment (alextremp) [inline src/openads-appnexus/AppNexusConnector.js]: the requestObj is not a named param in the prebidjs API reference, there's no sense to have it named here, it'll be better to send the requestObj as a whole parameters object, or parametrize the prebid client method with the 4 expected parameters, but not using requestObj as parameter :)

Comment (rafamarquezv) [reply src/openads-appnexus/AppNexusConnector.js]: @alextremp it is done. ^_^

**Review by rafamarquezv — APPROVED** [review]

**Review by alextremp — APPROVED** [review]
GOOD!

**Pull request merged**